l10n.js

```diff
@@ -197,7 +202,6 @@ document.webL10n = (function(window, document, undefined) {
       var dictionary = {};
 
       // token expressions
-      var reBlank = /^\s*|\s*$/;
```

Comment:
I was trimming the lines later on at processing so removed this. I dont *think* it would cause any issues with upgrading but have added it back in